h5writer/h5writer_single.py

```diff
@@ -45,6 +61,25 @@ def write_slice(self, data_dict):
         # Update of maximum index
         self._i_max = max([self._i, self._i_max])
 
+    def write_stack(self, stack: dict) -> None:
```

Comment:
We already use the term `stack` for the entire stack. How about naming this method `write_concatenated_slices` to be more clear?